### PR TITLE
Expand manager with two more operations

### PR DESCRIPTION
**Pull request by nionis**
Related to #142

### CODE REVIEW

**Comment by ausias-armesto** [comment]
I though that those utilities that you were building were going to be focused on Rpc-h, but here I see a function for registering a node, shouldn't this be somewhere else where core team can reuse it as well. @nionis

**Comment by nionis** [comment]
@ausias-armesto eventually yes, we need to collaborate so we can re-use these tools

File: devkit/manager/src/fund-via-wallet.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  });
+
+  const provider = new providers.JsonRpcProvider(providerStr);
+  const wallet = new Wallet(privateKey).connect(provider);
+
+  // check if wallet has enough balance to do transactions
+  const walletBalance = await wallet.getBalance();
+  if (walletBalance.lt(utils.parseEther("0.1"))) {
+    throw Error("Wallet balance is less than 0.1");
+  }
```
Comment (diegoalzate) [inline devkit/manager/src/fund-via-wallet.ts:38]: should we also check if wallet has enough hopr balance?

Comment (nionis) [reply devkit/manager/src/fund-via-wallet.ts:38]: added

**Review by diegoalzate — APPROVED** [review]

**Pull request merged**